### PR TITLE
Elzero task 1

### PR DESCRIPTION
**Pull request by mgkh286**
add list , image, link

### CODE REVIEW

File: src/elzero-task1/elzerotask1.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    <ul>
+      <li>HTML</li>
+      <li>CSS</li>
+      <li>JavaScript</li>
+      <ul>
+        <li>Vuejs</li>
+        <li>Reactjs</li>
+        <li>Angular</li>
+
+        <ol>
```
Comment (hossam-khalaf) [inline src/elzero-task1/elzerotask1.html]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ol

the ordered list starts from (4) , please fix it using the link above

**Review by hossam-khalaf — COMMENTED** [review]
Good Job

File: src/elzero-task1/elzerotask1.html (hunk 1 above)
Comment (mgkh286) [reply src/elzero-task1/elzerotask1.html]: done , thank you for helping me

**Review by hossam-khalaf — APPROVED** [review]
good job

**Pull request merged**